src/display/api.js

```diff
@@ -957,10 +957,41 @@ var PDFPageProxy = (function PDFPageProxyClosure() {
      */
     getTextContent: function PDFPageProxy_getTextContent(params) {
       params = params || {};
-      return this.transport.messageHandler.sendWithPromise('GetTextContent', {
+      var readableStream =
+        this.transport.messageHandler.sendWithStream('GetTextContent', {
```

Comment:
Nit: As indicated in https://github.com/mozilla/pdf.js/pull/8472#discussion_r119935306, please indent *this* line with two additional spaces (and let's use `let` instead when adding *new* code.)